addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2400,7 +2423,8 @@ options.registry.anchor = options.Class.extend({
             }
             this._setAnchorName(anchorName + n);
         }
-        return `${window.location.pathname}#${this.$target[0].id}`;
+        const pathName = this.isModal ? '' : `${window.location.pathname}`;
+        return pathName + `#${this.$target[0].id}`;
```

Comment:
Combining string concatenation and template strings in the same expression seems a bit schyzophrenic.

Reply:
I changed that